src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/index.ts

```diff
@@ -131,6 +132,11 @@ export const schemaComponents: IComponent[] = [
     Tag: AttachmentListComponent,
     Type: ComponentTypes.AttachmentList,
   },
+  {
+    name: 'NavComponent',
```

Comment:
As you can see all other components don't include "component" in the name.

I suggest renaming to `NavigationBar` to get consistency with `NavigationButtons`

Reply:
It makes sense